src/UIKit/UIApplication.cs

```diff
@@ -128,6 +128,26 @@ public static UIContentSizeCategory ToEnum (NSString key)
 			// Do not compare to Medium, it is the fallback
 			return UIContentSizeCategory.Medium;
 		}
+
+		public static NSString FromEnum (UIContentSizeCategory category)
```

Comment:
The generator can generate similar code in a consistent pattern automatically if you add `[Field ("...")]` to the `enum` values. Much easier to audit for missing values added in later OS versions.